### ISSUE TITLE
TOSCA 2.0 support

### ISSUE THREAD

**Issue by cdanger** [tliron/puccini-vscode#3]
Does this support TOSCA 2.0 or any plan to support it in a near future?

**Comment by tliron** [comment]
Yes, the plan is to track 2.0 support as it is implemented in Puccini.
